### PR TITLE
Add delegate to organizations and write specs for search functionality

### PR DESCRIPTION
**Pull request by DamirSvrtan**
PR for issue #112

### CODE REVIEW

**Comment by DamirSvrtan** [comment]
@ecrisostomo Please review the Pull request for the SAML integration since the testing Ruby versions in the .travis.yml are changed there. I would like to see that in the master branch so the other PR's are tested against proper ruby versions.

File: spec/client_spec.rb [hunk 1]
```diff
@@ -628,6 +626,41 @@
     end
   end
 
+  describe '#organization' do
```
Comment (ecrisostomo) [inline spec/client_spec.rb]: I was trying to do this on [this branch's file](https://github.com/stormpath/stormpath-sdk-ruby/blob/add-organizations-delegate/spec/client_spec.rb). Can you add all of the organization specs that I wrote there to see if they pass? They should, but they were not passing for me for some reason.

**Comment by DamirSvrtan** [comment]
Hey @ecrisostomo, I've added your test cases to my PR.

**Comment by ecrisostomo** [comment]
Looks good to merge!

**Pull request merged**